File: SoloLearn/Skee Ball.py
TICKET=12 # 1 ticket= 12 Points
try:
    points_scored=int(input('Points: '))
    gun_cost_by_ticket=int(input('cost: '))
    ticket_exchange=points_scored//TICKET
    if ticket_exchange>=gun_cost_by_ticket:
        print('Buy it!')
    else:
        print('Try again')
except:
    print('Invalid Number')

File: Java Code/Skee_Ball.java
import java.util.Scanner;

public class Skee_Ball {

	public static void main(String[] args) {
		final int SCALE=12;
		String result="";
		Scanner input= new Scanner(System.in);
		int points=input.nextInt();
		int cost=input.nextInt();
		input.close();
		int received_tickets=points/SCALE;
		if (received_tickets >= cost) result = "Buy it!";
		else result ="Try again";
		System.out.println(result);
	}

}
